sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_query_text.py

```diff
@@ -38,6 +38,7 @@ def test_query_text_llc(self, qna_account, qna_key):
                     "id": "doc2"
                 },
                 {
+                    # cspell:ignore groovin Olufsen
```

Comment:
We try not to add cspell ignores inline with the code. Would you mind adding these words to the cspell.json override section under questionanswering? Thanks!

Reply:
As you wish, madame! Thank you for showing me best practices; it's such a valuable asset for a beginner like me. Bless you and stay healthy Krista!

Reply:
No problem! Thanks for making the changes! I pushed a commit to fix the merge conflict, once that's green in CI I will approve and merge! 😸